app/views/idv/how_to_verify/show.html.erb

```diff
@@ -11,57 +11,69 @@
     ) do |f|
 %>
   <div role="group" aria-labelledby="how-to-verify-info">
-    <%= f.radio_button(
+    <%= f.hidden_field(
           :selection,
-          Idv::HowToVerifyForm::REMOTE,
-          class: 'usa-radio__input usa-radio__input--tile',
+          value: Idv::HowToVerifyForm::REMOTE,
         ) %>
     <%= f.label(
           :selection_remote,
-          class: 'usa-radio__label usa-radio__label--illustrated usa-radio__label-illustrated-large',
         ) do %>
-          <%= image_tag asset_url('idv/remote.svg'),
-                        width: 88,
-                        height: 92,
-                        class: 'usa-radio__image',
-                        alt: t('image_description.laptop_and_phone')
-          %>
-          <div class="usa-radio__label--text">
-            <span class="usa-tag usa-tag--informative">
-              <%= t('doc_auth.tips.most_common') %>
-            </span>
-            <h2 class="h3"><%= t('doc_auth.headings.verify_online') %></h2>
-            <span class="usa-radio__label-description">
-              <p><%= t('doc_auth.info.verify_online_instruction') %></p>
-              <p><%= t('doc_auth.info.verify_online_description') %></p>
-            </span>
+          <div class="grid-row border-bottom border-primary-light">
+            <div classe="tablet:grid-col">
+              <%= image_tag asset_url('idv/remote.svg'),
+                            width: 88,
+                            height: 88,
+                            class: 'margin-right-1',
+                            alt: t('image_description.laptop_and_phone')
+              %>
+            </div>
+            <div class="tablet:grid-col">
+              <h2 class="h3 margin-top-0"><%= t('doc_auth.headings.verify_online') %></h2>
+              <div>
+                <p><%= t('doc_auth.info.verify_online_instruction') %></p>
+                <p><%= t('doc_auth.info.verify_online_description') %></p>
+              </div>
+              <%= f.submit t('forms.buttons.continue_remote'), class: 'display-block margin-y-5' %>
+            </div>
           </div>
         <% end %>
-    <%= f.radio_button(
+  </div>
+<% end %>
+<%= simple_form_for(
```

Comment:
I'm surprised this isn't triggering a [form labeling accessibility test](https://github.com/18F/identity-idp/blob/4aa6aeb7b3db60c02c84107e1b577716e3001862/spec/support/matchers/accessibility.rb#L143-L171) that was added in #9508, since my understanding is that if we have more than one form on a page, they need to be labelled. Does that apply here?

Reply:
added aria-labels as well as a test for the expectation in the code you linked